packtools/sps/validation/journal_meta.py

```diff
@@ -69,65 +69,79 @@ def validate_issn(self, issns_dict):
-                'response': 'OK' if is_valid else 'ERROR',
-                'expected_value': expected_value,
-                'got_value': self.journal_acronym.text,
-                'message': 'Got {} expected {}'.format(self.journal_acronym.text, expected_value),
-                'advice': None if is_valid else 'Provide an acronym value as expected: {}'.format(expected_value)
-            }
-        ]
+        yield format_response(
+            title='Journal acronym',
+            parent='article',
+            parent_id=None,
+            parent_article_type=self.xmltree.get("article-type"),
+            parent_lang=self.xmltree.get("{http://www.w3.org/XML/1998/namespace}lang"),
+            item='journal-id',
+            sub_item='@journal-id-type="publisher-id"',
+            validation_type='value',
+            is_valid=is_valid,
+            expected=expected_value,
+            obtained=self.journal_acronym.text,
+            advice='Mark journal acronym value with <journal-meta><journal-id journal-id-type="publisher-id">{}</journal-id>'.format(expected_value),
+            data={'acronym': self.journal_acronym.text},
+            error_level=error_level,
+        )
 
 
 class TitleValidation:
     def __init__(self, xmltree):
         self.xmltree = xmltree
         self.journal_titles = Title(xmltree)
 
-    def journal_title_validation(self, expected_value):
+    def journal_title_validation(self, expected_value, error_level="CRITICAL"):
         if not expected_value:
             raise ValidationJournalMetaException('Function requires a value to journal title')
         is_valid = self.journal_titles.journal_title == expected_value
-        return [
-            {
-                'title': 'Journal title element validation',
-                'xpath': './journal-meta/journal-title-group/journal-title',
-                'validation_type': 'value',
-                'response': 'OK' if is_valid else 'ERROR',
-                'expected_value': expected_value,
-                'got_value': self.journal_titles.journal_title,
-                'message': 'Got {} expected {}'.format(self.journal_titles.journal_title, expected_value),
-                'advice': None if is_valid else 'Provide a journal title value as expected: {}'.format(expected_value)
-            }
-        ]
+        yield format_response(
+            title='Journal title',
+            parent='article',
+            parent_id=None,
+            parent_article_type=self.xmltree.get("article-type"),
+            parent_lang=self.xmltree.get("{http://www.w3.org/XML/1998/namespace}lang"),
+            item='journal-title-group',
+            sub_item='journal-title',
+            validation_type='value',
+            is_valid=is_valid,
+            expected=expected_value,
+            obtained=self.journal_titles.journal_title,
+            advice='Mark journal title value with <journal-meta><journal-title-group><journal-title>',
```

Comment:
@Rossi-Luciano remover `<journal-meta>`e adicionar ao final da mensagem `in <journal-meta>`

Reply:
OK, @robertatakenaka